pkg/client/query/sharedquerier.go

```diff
@@ -89,7 +89,7 @@ func (sq *sharedQuerier) GetParams(ctx context.Context) (*sharedtypes.Params, er
 // GetClaimWindowOpenHeight returns the block height at which the claim window of
 // the session that includes queryHeight opens.
 //
-// TODO_MAINNET(#543): We don't really want to have to query the params for every method call.
+// TODO_MAINNET_MIGRATION(@bryanchriswhite, #543): We don't really want to have to query the params for every method call.
```

Comment:
Shouldn't these be @red-0ne, given that he's working on historical params (this would be consisten with the diff in x/proof/types/shared_query_client.go, below)?